src/js/components/Widgets/ShowMoreButtons.jsx

```diff
@@ -44,7 +51,13 @@ class ShowMoreButtons extends React.Component {
         userCohort: VoterStore.getAnalyticsUserCohort(),
         voterWeVoteId: VoterStore.getVoterWeVoteId(),
       },
+      officeDetails: {
```

Comment:
Instead of always adding officeDetails to dataLayerObject, let only add it to dataLayerObject if the officeWeVoteId prop exists.